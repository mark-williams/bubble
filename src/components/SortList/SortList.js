import React from 'react';
import PropTypes from 'prop-types';

const SortList = ({ list }) => (
  <div className="sortList">
    {list.map(n => <div className="sortItem">{n}</div>)}
  </div>
);

SortList.propTypes = {
  list: PropTypes.arrayOf(PropTypes.object),
};

export default SortList;
