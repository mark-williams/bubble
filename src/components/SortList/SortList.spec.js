import React from 'react';
import renderer from 'react-test-renderer';
import SortList from './SortList';

const testItems = [
  { id: 1000, number: 71 },
];

describe('SortList', () => {
  it('snapshot', () => {
    const tree = renderer.create(
      <SortList list={testItems} />,
    );
    expect(tree).toMatchSnapshot();
  });
});
