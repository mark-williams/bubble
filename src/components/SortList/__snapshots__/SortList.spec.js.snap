// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortList snapshot 1`] = `
<div
  className="sc-bwzfXH icOaIC"
>
  <div
    className="sc-bdVaJa cHazHO"
  >
    71
  </div>
</div>
`;
